### ISSUE TITLE
How to change base path for etherpad helm chart

### ISSUE THREAD

**Issue by massih10** [nicholaswilde/helm-charts#58]
Hi , is it possible to deploy the etherpad helm chart with an ingress rule under a different path, say https://somedomain/etherpad ?

**Comment by nicholaswilde** [comment]
Yes, it is possible. See https://hackernoon.com/expose-your-kubernetes-service-from-your-own-custom-domains-cc8a1d965fc for an example. If you have more questions, you may join the k8s@home discord channel, https://discord.gg/sTMX7Vh.
